components/broadcast_talent_table/commons/broadcast_talent_table.lua

```diff
@@ -341,13 +335,6 @@ end
 ---@return string
 function BroadcastTalentTable:_tierDisplay(tournament)
 	local tier, tierType, options = Tier.parseFromQueryData(tournament)
-	if not tier then
-		--on sc2 sometimes there are broadcaster cards on series pages
-		--for those the additional tournament data is stored in extradata
-		--including tier and tier type
-		tier, tierType, options = Tier.parseFromQueryData(tournament.extradata)
-	end
-
 	assert(tier, 'Broadcaster event with unset or invalid liquipedia tier: ' .. tournament.pagename)
```

Comment:
```suggestion
	assert(Tier.isValid(tier, tierType), 'Broadcaster event with unset or invalid liquipedia tier: ' .. tournament.pagename)
```

so it also catches empty strings on wikiss where they are not allowed (they are valid on smash)